Makefile

```
@@ -1,7 +1,10 @@
+GITREV=`git describe --tags`
```

Maybe something like this instead, so we can keep `softwareVer` without the **v** included? (It's used in other parts of the app where we don't want the **v**, including in filenames of the archives we release.)

```suggestion
GITREV=`git describe --tags | cut -c 2-`
```